_posts/2021-12-31-trino-2021-a-year-of-growth.md

```diff
@@ -0,0 +1,348 @@
+ </a>
+</p>
+
+## Community growth
+
+While we are on the subject, lets talk about the health of the Trino community.
+As mentioned the GitHub stargazers trajectory skyrocketed. At the time of 
+writing, we just hit 4,600 stargazers on GitHub. This means, we have added over
+3,000 stargazers in the last year, a 187% increase.
+
+Aside from GitHub, we have seen a 47% growth of [the Trino Twitter](https://twitter.com/trinodb) 
+followers this year. [The Trino Slack community](https://trino.io/slack.html),
+where a large amount of troubleshooting and development discussions occur, saw a
+75% growth nearing 6,000 members. Finally, [the Trino YouTube channel](https://www.youtube.com/c/TrinoDB)
+has seen an impressive 280% growth in subscribers as the [Trino Community Broadcast](/broadcast/)
+has grown in the number of helpful videos for community members.
+
+While stargazers, subscribers, and followers tell the story of the growing 
+awareness of the Trino project with the new name, what about the actual rate of
+development on the project?
+
+At the start of the year, there were 21,924 commits. This year, we pushed 3,679 
+commits to the repository,sitting at over 25,600 now. Looking at the graph, this
+keeps us pretty consistent with 2020s throughput.
+
+<p align="center">
+ <img align="center" width="75%" src="/assets/blog/2021-review/commits.png"/>
+</p>
+
+With the project's trajectory displayed in numbers, let's examine the top 
+features that landed in Trino this year.
+
+## Features
+
+Here's a high-level list of the most exciting features that made their way into
+Trino in 2021. For details and to keep up you can check out the [release notes]({{site.url}}/docs/current/release.html).
+
+### Improvements to ANSI SQL support
+
+SQL language support is crucial for the increasing complexities of queries and 
+usage of Trino. In 2021 we added numerous new language features and 
+improvements:
+
+* [`RANGE`]({{site.url}}/blog/2021/03/10/introducing-new-window-features.html#new%20features)
+keyword for usage within a `WINDOW` function.
+* [`MATCH_RECOGNIZE`]({{site.url}}/blog/2021/05/19/row_pattern_matching.html)
+a feature that allows for complex analysis across multiple rows. To learn more 
+about this feature watch [the Community Broadcast show](/episodes/23.html).
+* Aggregate functions in for `MATCH_RECOGNIZE`.
+* `TRUNCATE TABLE` which allows for a more efficient delete.
+* `DENY` statement, which enables you to remove a user or groups access via SQL.
+* `IN catalog` clause to `CREATE ROLE`, `DROP ROLE`, `GRANT ROLE`, 
+`REVOKE ROLE`, and `SET ROLE` to specify the target catalog of the statement 
+instead of using the current session catalog.
+* [`listagg()`]({{site.url}}/docs/current/functions/aggregate.html#listagg)
+function returns concatenated values seperated by a specified separator.
+* [`SET TIME ZONE`]({{site.url}}/docs/current/sql/set-time-zone.html) keywords.
```

Comment:
Describe briefly what this does: "SET TIME ZONE, to set the current time zone for the session."